### DISCUSSION TITLE
RabbitMQ -> Erlang erlsrv: Error, service parameter keys nonexistent.

### DISCUSSION THREAD

**Discussion by steveOgil** [rabbitmq/rabbitmq-server#4360] [Questions]
Hello,
Using RabbitMQ 3.9.13 and Erlang 24.2.2 and OpenSSL3.0.1. We have seen this error quite a few times starting with RabbitMQ 3.8.0, here is the latest (3.9.13):

```
HOMEDRIVE is set to C: by default
SystemDrive is set to C:
HOMEDRIVE is now set to C:
ERLANG_HOME is set to C:\Program Files\erl-24.2.2
RABBITMQ_BASE is set to C:\ProgramData\Genetec\RabbitMQ
RABBITMQ_CONFIG_FILE is set to C:\ProgramData\Genetec\RabbitMQ\config\rabbitmq
RABBITMQ_ADVANCED_CONFIG_FILE is set to C:\ProgramData\Genetec\RabbitMQ\config\advanced
RABBITMQ_CONSOLE_LOG is set to new
VerifyRabbitMqServiceExists: RabbitMQ service exists
RemoveRabbitMqService: Path to RabbitMQ service bat file: C:\Program Files\RabbitMQ Server\rabbitmq_server-3.9.13\sbin\rabbitmq-service.bat argument is remove
Running C:\Program Files\RabbitMQ Server\rabbitmq_server-3.9.13\sbin\rabbitmq-service.bat with argument remove returned 1 - C:\Program Files\erl-24.2.2\erts-12.2.1\bin\erlsrv: Error, service parameter keys nonexistent.
```

can you suggest any information that might help me resolve these type of errors?

Cheers,

Steve

**Comment by michaelklishin** [comment]
See server logs for extra clues, make sure you don't have multiple Erlang installations, reinstall the service using *an administrator account*. 

See  [Windows Quirks](https://rabbitmq.com/windows-quirks.html).

**Comment by michaelklishin** [comment]
@steveOgil you have received a response here and never engaged to provide any log data. Please stop asking the same questions by contacting our team via different channels and in private. If you have any logs, service output or other clues, please post them here.

If you have tried reinstalling the service **using an administrator account**, post exactly what you did and your findings here.
We cannot help you beyond the above suggestions without any additional information, and posting the same question all over the Internet is not going to be very productive.
